init/pkg/create-aerospike-conf.go

```diff
@@ -35,14 +35,34 @@ func (initp *InitParams) createAerospikeConf() error {
 	}
 
 	confString = strings.ReplaceAll(confString, "ENV_NODE_ID", initp.nodeID)
-	confString = initp.substituteEndpoint(initp.networkInfo.networkPolicy.AccessType, access, confString)
-	confString = initp.substituteEndpoint(initp.networkInfo.networkPolicy.AlternateAccessType, alternateAccess, confString)
+	confString = initp.substituteEndpoint(
```

Comment:
any reason of changing the name of initp in above files and not here and elsewhere?